modules/core/02-client/client/cli/tx.go

```diff
@@ -17,6 +14,9 @@ import (
 	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/pkg/errors"
```

Comment:
Random question: Any ideas why this is used over std lib `errors`? I think that repo is archived. cc @colin-axner

Reply:
No idea. Want to open an issue?

Reply:
Sure. I can do! :)